src/app/templates/sidebar/sidebar.component.less

```diff
@@ -37,6 +37,7 @@
             letter-spacing: 0.02em;
             text-transform: uppercase;
             color: @gray1;
+            word-break: normal;
```

Comment:
Please add specific modifications to this style where you import (m6.info.component.less) it and not in the main file.

Reply:
done